src/DoubleComparator.php

```diff
@@ -1,56 +0,0 @@
-<?php
-/*
- * This file is part of sebastian/comparator.
- *
- * (c) Sebastian Bergmann <sebastian@phpunit.de>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-namespace SebastianBergmann\Comparator;
-
-/**
- * Compares doubles for equality.
- */
-class DoubleComparator extends NumericComparator
```

Comment:
The removal of `DoubleComparator` is a BC break. We should keep the class around and just do not use it anymore.

Reply:
done and marked as deprecated, when merging to `main`, `DoubleComparator` and `NumericComparatorTest` classes/files should be deleted